docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
+ * Mac OSX 10.9, 10.10 and 10.11
+ * Linux: Any system with Python 2.7
+ * Debian/Raspberry Pi packages: Wheezy or later
+ * Ubuntu packages: 14.04, 15.10, 16.04 - anything that's *not* end-of-life.
+
+
 Supported Browsers
 ------------------
-KA Lite is currently *not* supported on Internet Explorer version 8 or lower. You must use IE9 or later.
 
-You can also use KA Lite on browsers like Firefox, Chrome, and Safari!
+ * IE9+
+ * Firefox
```

Comment:
We should specify versions we know that are working at least. I debug stuff on Firefox version 45 and Safari 9. Let's specify that these versions and later are supported.